TestChainExportImportFull is flaky

```
2020-10-22T22:41:24.218Z	�[31mERROR�[0m	statetree	state/statetree.go:205	loading hamt node bafy2bzaceb4ilrzlqrxgzrzhteoxul3q5etprrpb5rlbzjpnhtnnwvz2jchwa failed: failed to load hamt node: blockstore: block not found
    store_test.go:158: load state tree: failed to load hamt node: blockstore: block not found
--- FAIL: TestChainExportImportFull (1.88s)
```

--- a/chain/store/store_test.go
+++ b/chain/store/store_test.go
@@ -126,7 +126,7 @@ func TestChainExportImportFull(t *testing.T) {
 	}
 
 	buf := new(bytes.Buffer)
-	if err := cg.ChainStore().Export(context.TODO(), last, 100, false, buf); err != nil {
+	if err := cg.ChainStore().Export(context.TODO(), last, last.Height(), false, buf); err != nil {
 		t.Fatal(err)
 	}
 

